lib/managers/folders.js

```diff
@@ -321,6 +322,26 @@ Folders.prototype.getTrashedFolder = function(folderID, qs, callback) {
 	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
 };
 
+/**
+ * Retrieves the files and/or folders that have been moved to the trash.
+ *
+ * API Endpoint: '/folders/trash/items'
+ * Method: GET
+ *
+ * @param {?Object} qs - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param {Function} callback - Called with nothing if successful, error otherwise
```

Comment:
This comment needs to be updated; the callback is called with the items in the trash when successful